exercise_perseus_render/assets/src/vue/perseus.vue

```diff
@@ -457,6 +464,9 @@
     .paragraph
       padding: 4px
 
+    .fixed-to-responsive
+      display: inline-block
+
```

Comment:
> All css rules that override the original is in one file, perseus.vue. It's not that hard to maintain. And we control the perseus update frequency. I think the gain overweights the cost.

It's not just when we update perseus, it's when we get new questions from KA. And have you checked it works correctly for all the currently existing 25,000+ questions?

(I'm assuming the styles are affecting more than the outer shell, as there's stuff about image offsets, etc)